src/cloud-api-adaptor/podvm/README.md

```diff
@@ -43,36 +43,27 @@ The builder image packages the cloud-api-adaptor and Kata Containers sources as
 the binaries (e.g. *kata-agent* and *agent-protocol-forwarder*) that should be installed in the podvm image.
 
 The builder image is agnostic to cloud providers in the sense that one can be used to build for multiple providers, however it is
-dependent on the Linux distribution the image is built for. Therefore, in this directory you will find dockerfiles for each supported distributions, which are currently Ubuntu 20.04 ([Dockerfile.podvm_builder](./Dockerfile.podvm_builder)), and RHEL 9 ([Dockerfile.podvm_builder.rhel](./Dockerfile.podvm_builder.rhel)).
-
-As an example, to build the builder image for Ubuntu, run:
+dependent on the Linux distribution the image is built for. Therefore, in this directory you will find dockerfiles for each
+supported distributions, which are currently Ubuntu 20.04 ([Dockerfile.podvm_builder](./Dockerfile.podvm_builder)),
+Fedora 39 ([Dockerfile.podvm_builder.fedora](./Dockerfile.podvm_builder)) and RHEL 9
+([Dockerfile.podvm_builder.rhel](./Dockerfile.podvm_builder.rhel)).
 
+You can create the builder image using the make target under the `src/cloud-api-adaptor` folder by running:
 ```bash
-$ docker build -t podvm_builder \
-         -f Dockerfile.podvm_builder .
+$ make podvm-builder
```

Comment:
Hi @stevenhorsman !

Make should be executed from parent folder, so maybe `make -C .. podvm-builder`

Reply:
So in the line above I mentioned having to run it in the 
> `src/cloud-api-adaptor` folder
but I can swap to this approach if you prefer (and I'll update Beraldo's recent change to match)

Reply:
that's ok, thanks Steve!

Reply:
I had to rebase to resolve conflicts, so I updated this. This will change when we ditch packer for mkosi anyway!